src/form.js

```diff
@@ -188,7 +187,7 @@ define([
         }
         return that;
     };
-    var INSTANCE_REGEXP = /^instance\((['"])([^'"]+)\1\)/i;
+    var INSTANCE_REGEXP = /(^)instance\((['"])([^'"]+)\2\)/i;
```

Comment:
This group is functionally unnecessary in this regexp. I added it to make the relative position and number of groups match the [regexp below on line 341](https://github.com/dimagi/Vellum/pull/468/files#diff-62acef2590804164cb77b4b5aced00ebR341), allowing either to be used in `updateInstanceQuery`. Notice the replacement string on [line 344](https://github.com/dimagi/Vellum/pull/468/files#diff-62acef2590804164cb77b4b5aced00ebR344) references it.